### PR TITLE
Added PHG Campaign Token support.

### PR DESCRIPTION
**Pull request by claesjacobsson**

### CODE REVIEW

File: iTunesSearch/ItunesSearch.m [hunk 1]
```diff
@@ -96,6 +97,10 @@ - (void)performApiCallForMethod:(NSString*)method
     if (self.affiliateToken && self.affiliateToken.length > 0)
         newParams[@"at"] = self.affiliateToken;
 
+    // Add campaign token if supplied
+    if (self.campaignToken && self.campaignToken.length > 0)
```
Comment (eightytwo) [inline iTunesSearch/ItunesSearch.m]: Would you mind using curly braces in this if statement? Thanks!

Comment (claesjacobsson) [inline iTunesSearch/ItunesSearch.m]: I could absolutely do that, but don't you want me to follow the code style? See the lines just before... Just checking :)

Comment (claesjacobsson) [inline iTunesSearch/ItunesSearch.m]: I see now that curly braces was probably supposed to be used there to. I'' make a new commit.

Comment (eightytwo) [inline iTunesSearch/ItunesSearch.m]: Good catch. Would you mind adding braces to that if statement too?

Comment (claesjacobsson) [inline iTunesSearch/ItunesSearch.m]: Yep.

**Comment by eightytwo** [comment]
Thanks for the PR :)

**Comment by claesjacobsson** [comment]
Thank you!

2015-05-12 11:50 GMT+02:00 Piers <notifications@github.com>:

**Comment by eightytwo** [comment]
Just a heads up, your commit is included in v1.0.0 which is available via cocoapods.

**Comment by claesjacobsson** [comment]
I was hoping for that :) Thanks!

2015-05-12 12:06 GMT+02:00 Piers <notifications@github.com>:

**Pull request merged**